src/main/java/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue.java

```diff
@@ -149,7 +153,7 @@ public String toString() {
         valueStr.append("(MatrixCombinationsParameterValue) " + getName()+"\n");
         for (int i=0; i< values.length; i++)
         {
-            valueStr.append(String.format("%s:%s\n",confs[i],values[i]));
+            valueStr.append(String.format("%s:%s%n",confs[i],values[i]));
```

Comment:
<BugInstance type='VA_FORMAT_STRING_USES_NEWLINE' priority='Normal' category='BAD_PRACTICE' message='Format string should use %n rather than \n in hudson.plugins.matrix_configuration_parameter.MatrixCombinationsParameterValue.toString()' lineNumber='152'/>